scripts/create_workspace.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -e
+
+# Read the name of new workspace from the command line.
+if [ $# == 1 ]; then
+    NEW_WS_NAME_UNDERSCORE="$(echo $1 | sed "s/-/_/g")"
+    NEW_WS_NAME_HYPHEN="$(echo $1 | sed "s/_/-/g")"
+else
+    echo "Error: Please specify the name of new workspace."
+    echo "Usage: $0 <new_workspace_name>"
+    exit 1
+fi
+
+# Get the directory of this script.
+# Reference: 
+# - https://stackoverflow.com/questions/59895/how-do-i-get-the-directory-where-a-bash-script-is-located-from-within-the-script
```

Comment:
Consider use the shortened URL by the `Share` feature of Stack Overflow:

https://stackoverflow.com/q/59895